products/dev_alternate_products.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Plugins Targeting IntelliJ Platform-Based IDEs 
+---
+
+This page is an introduction to configuring Gradle plugin projects that target IDEs based on the [IntelliJ Platform](/intro/intellij_platform.md), but are not IntelliJ IDEA.
+All of the Gradle configuration attributes mentioned here are discussed in-depth on the [Configuring Gradle for IntelliJ Platform Plugins](/tutorials/build_system/gradle_guide.md) page.
+
+Information common to projects targeting products other than IntelliJ IDEA is introduced here. 
+Details particular to an IntelliJ Platform-based IDE is described on the individual product pages in Part VIII.
+
+* bullet list
+{:toc}
+
+## Introduction 
+Plugins can extend or add functionality to IDEs based on the IntelliJ Platform. 
+This extensibility applies to a variety of JetBrains' IDEs, such as Android Studio, WebStorm, PhpStorm, and PyCharm.
+Plugins can be written in Kotlin or Java, or a mix of both, and are created using IntelliJ IDEA.
```

Comment:
not sure we need line 17/18 here

Reply:
16 can go.
17 - Mention of Kotlin was requested by Seb when reviewing the page for Android Studio, so I think it is good to "promote" the sentence to this page.